py-scripts/lf_webpage.py

```diff
@@ -177,73 +306,69 @@ def phantom_check(self):
                                 else:
                                     eid_list.append(b['eid'])
                                     mac_list.append(b['hw version'])
-                                    # self.hostname_list.append(b['eid']+ " " +b['hostname'])
                                     devices_available.append(b['eid'] + " " + 'Mac' + " " + b['hostname'])
                             else:
                                 eid_list.append(b['eid'])
                                 android_list.append(b['hw version'])
                                 # self.username_list.append(b['eid']+ " " +b['user'])
                                 devices_available.append(b['eid'] + " " + 'android' + " " + b['user'])
-        # print("hostname list :",self.hostname_list)
-        # print("username list :", self.username_list)
-        # print("Available resources in resource tab :", devices_available)
-        # print("eid_list : ",eid_list)
         # All the available resources are fetched from resource mgr tab ----
 
-        response_port = self.local_realm.json_get("/port/all")
-        # print(response_port)
+        response_port = self.json_get("/port/all")
         mac_id1_list = []
         for interface in response_port['interfaces']:
             for port, port_data in interface.items():
-                if (not port_data['phantom'] and not port_data['down'] and port_data['parent dev'] == "wiphy0" and port_data['alias'] != 'p2p0'):
+                if not port_data['phantom'] and not port_data['down'] and port_data['parent dev'] == "wiphy0" and port_data['alias'] != 'p2p0':
                     for id in eid_list:
-                        if (id + '.' in port):
+                        if id + '.' in port:
                             original_port_list.append(port)
-                            port_eid_list.append(str(self.name_to_eid(port)[0]) + '.' + str(self.name_to_eid(port)[1]))
-                            mac_id1_list.append(str(self.name_to_eid(port)[0]) + '.' + str(self.name_to_eid(port)[1]) + ' ' + port_data['mac'])
-        # print("port eid list",port_eid_list)
+                            port_eid_list.append(str(LFUtils.name_to_eid(port)[0]) + '.' + str(LFUtils.name_to_eid(port)[1]))
+                            mac_id_list1.append(str(LFUtils.name_to_eid(port)[0]) + '.' + str(LFUtils.name_to_eid(port)[1]) + ' ' + port_data['mac'])
```

Comment:
This is the type of code that whitespace changes included in logic changes makes difficult for me to review. Nothing changed here that I can tell, but I have to use extra review time to make that determination. No change needed as of now

Reply:
Changing my mind slightly, it would be welcome if you make these whitespace changes in a separate commit. Not required but it would really help for review